frontend/packages/vkt/src/components/clerkExamEvent/overview/ClerkExamEventEnrollmentListingRow.tsx

```diff
@@ -0,0 +1,83 @@
+import { TableCell, TableRow } from '@mui/material';
+import { Text } from 'shared/components';
+import { DateUtils } from 'shared/utils';
+
+import { useClerkTranslation } from 'configs/i18n';
+import { Enrollment } from 'interfaces/clerkExamEvent';
+
+const examCodes = {
+  oralSkill: 'ST',
```

Comment:
Tästä tosiaan itse weeklyssä mainitsinkin ja musta tähän ei selkeää lopullista päätöstä taidettu saada? Mutta itse näkisin että taitoja (ST, TE, YM) ei tarvitsisi tutkinnon tarkastelusivulla ilmoittautumisten osalta näyttää vaan riittää pelkästään näyttää sopivassa loogisessa järjestyksessä valitut osakokeet. Näin virkailija voi havaita, paljonko osallistujia on mihinkin osakokeisiin ja voi tuon puolesta katsoa, josko esim. jonossa on joku ilmoittautuja, joka mahtuisi mukaan tutkintoon jos kaikki eivät suorita jokaista osakoetta.

Valitsin itse ilmoittautumisflowhon toistaiseksi osakokeiden järjestykseksi muistaakseni (PU, PY, KI, TY). Sopivaa järjestystä ei ole taidettu virkailijoiden kanssa sopia, Figma-leiskoissa oli joku toinen järjestys mikä ehkä myös alustava.